django/db/models/constraints.py

```diff
@@ -79,16 +85,18 @@ class Deferrable(Enum):
 class UniqueConstraint(BaseConstraint):
     def __init__(
         self,
-        *,
-        fields,
```

Comment:
We cannot change the signature that much, it's backward incompatible. IMO it's acceptable to make `fields` and `name` arguments optional:
```python
    def __init__(
        self,
        *expressions,
        fields=(),
        name=None,
        ...
    ):
        if not name:
            raise ValueError('An unique constraint must be named.')
        ```
```

Reply:
Yikes, I was a bit too fast there. Thanks.